exchange/exchange.go

```diff
@@ -52,30 +50,26 @@ func NewExchange(client *http.Client, cache prebid_cache_client.Client, cfg *con
 	e := new(exchange)
 
 	e.adapterMap = newAdapterMap(client, cfg)
-	e.adapters = make([]openrtb_ext.BidderName, 0, len(e.adapterMap))
 	e.cache = cache
 	e.cacheTime = time.Duration(cfg.CacheURL.ExpectedTimeMillis) * time.Millisecond
-	for a, _ := range e.adapterMap {
-		e.adapters = append(e.adapters, a)
-	}
 	e.m = registry
 	return e
 }
 
 func (e *exchange) HoldAuction(ctx context.Context, bidRequest *openrtb.BidRequest, usersyncs IdFetcher) (*openrtb.BidResponse, error) {
 	// Slice of BidRequests, each a copy of the original cleaned to only contain bidder data for the named bidder
-	cleanRequests, errs := cleanOpenRTBRequests(bidRequest, e.adapters, usersyncs, e.m)
+	cleanRequests, aliases, errs := cleanOpenRTBRequests(bidRequest, usersyncs, e.m)
 	// List of bidders we have requests for.
 	liveAdapters := make([]openrtb_ext.BidderName, len(cleanRequests))
 	i := 0
-	for a, _ := range cleanRequests {
+	for a := range cleanRequests {
 		liveAdapters[i] = a
 		i++
 	}
 	// Randomize the list of adapters to make the auction more fair
 	randomizeList(liveAdapters)
 	// Process the request to check for targeting parameters.
-	var targData *targetData = nil
+	var targData *targetData
```

Comment:
I know the old and the new lines are functionally the same. Is there any advantage to not explicitly declaring the value of the variable to be the null value?

Reply:
not that I know of, no.

I've been trying out visual studio and it complains about those lines... I think cause it runs [golint](https://github.com/golang/lint) in the background. I really have no preference either way.